man/sd_journal_open.xml

```diff
@@ -29,6 +29,8 @@
     <refname>SD_JOURNAL_SYSTEM</refname>
     <refname>SD_JOURNAL_CURRENT_USER</refname>
     <refname>SD_JOURNAL_OS_ROOT</refname>
+    <refname>SD_JOURNAL_ALL_NAMESPACES</refname>
```

Comment:
For compatibility, will `journalctl -f` work as it does today, effectively looking at all namespaces?

Reply:
Yes and no. It will work as it does today, and show only the "default" namespace. If you want to to look at other namespaces, you need to specify that. the man page addition for the --namespace= switch should make that clear.

Since namespaces are about isolation of log streams I am very sure we should default to "do not merge log streams", and only merge them if that's explicitly requested. Otherwise we'd defeat the purpose of namespace somewhat...